kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/SessionManager.java

```diff
@@ -165,7 +163,8 @@ public void createEnvironment() {
                         resources.addAll(resourceInstaller.install(configUrl));
                     }
                 } else {
-                    log.warn("Did not find any kubernetes configuration.");
+                    log.warn(
+                        "Did not find any kubernetes configuration. If you are using fabric8-maven-plugin, ensure `mvn package fabric8:build` is run first.");
```

Comment:
I believe `mvn package fabric8:build` is not generating k8s/openshift configuration files, where it's building docker image. I couldn't find generated resources in target. Refer [build_log](https://gist.github.com/aa1611fc8e9fc46e41a574c6b0b0d97a).

When I tried `mvn package fabric8:resource`, then it created resources configuration. Refer [resource_log](https://gist.github.com/727912faa6872b4ad641fb324cee2118
)